### DISCUSSION TITLE
[转 QQ 群友] 被 Notron 360 订阅检测到威胁

### DISCUSSION THREAD

**Discussion by SuoHouDaiShi** [Meloong-Git/PCL#3220] [讨论]
![图片](https://github.com/Hex-Dragon/PCL2/assets/119282947/66d64314-4a1d-4a5c-bc08-1e4c84276393)
原 Issue： #3219

**Comment by allMagicNB** [comment]
看看检测到的病毒类型？

**Reply by SuoHouDaiShi** [reply]
![6{5K%@VA0`TT3(M}C PS${9](https://github.com/Hex-Dragon/PCL2/assets/119282947/649b56c4-b1b3-4884-ae7d-9d3aba0e57da)
这是 Notron 360 第一次报毒的图，具体是啥不知道，或许可以看看[这里？](https://www.virustotal.com/gui/file/608eb10319f73d7e7ad012a47fc83da722a359b12aa73eb29092245f7540f6f4)

**Reply by allMagicNB** [reply]
> ![6{5K%@VA0`TT3(M}C PS${9](https://github.com/Hex-Dragon/PCL2/assets/119282947/649b56c4-b1b3-4884-ae7d-9d3aba0e57da)
> 这是 Notron 360 第一次报毒的图，具体是啥不知道，或许可以看看[这里？](https://www.virustotal.com/gui/file/608eb10319f73d7e7ad012a47fc83da722a359b12aa73eb29092245f7540f6f4)

~~诺顿能量橡皮~~
